src/Symfony/Component/Encryption/Ciphertext.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Encryption;
+
+use Symfony\Component\Encryption\Exception\DecryptionException;
+use Symfony\Component\Encryption\Exception\MalformedCipherException;
+
+/**
+ * This class is responsible for the payload API.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @internal
+ */
+class Ciphertext implements \Stringable
+{
+    /**
+     * @var string algorithm for the encryption
+     */
+    private $algorithm;
```

Comment:
```suggestion
    private string $algorithm;
```

wdyt?
I've seen some PRs targeting v6 adding this kind of code for existing codebase,
thus maybe adding it directly to new one is a good option :)

Reply:
Good idea. I've update the PR